# HG changeset patch
# User Rik <rik@octave.org>
# Date 1503783600 25200
#      Sat Aug 26 14:40:00 2017 -0700
# Node ID 58435a400d4eee7c8cdb8020ae9b632c61d464b0
# Parent  9c97909a6c9479596331207eb3d85fdcd0800883
Document that fread 'float' size is 32 bits (bug #41672).

* file-io.cc (Ffread): Remove line stating 'float' is system-dependent and put
'float' in the table entry for 'single' which says the result is 32 bits.

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -2363,16 +2363,17 @@ 16-bit unsigned integer.
 
 @item @qcode{"uint32"}
 32-bit unsigned integer.
 
 @item @qcode{"uint64"}
 64-bit unsigned integer.
 
 @item  @qcode{"single"}
+@itemx @qcode{"float"}
 @itemx @qcode{"float32"}
 @itemx @qcode{"real*4"}
 32-bit floating point number.
 
 @item  @qcode{"double"}
 @itemx @qcode{"float64"}
 @itemx @qcode{"real*8"}
 64-bit floating point number.
@@ -2397,18 +2398,16 @@ Unsigned short integer (size is platform
 @item  @qcode{"uint"}
 @itemx @qcode{"unsigned int"}
 Unsigned integer (size is platform dependent).
 
 @item  @qcode{"ulong"}
 @itemx @qcode{"unsigned long"}
 Unsigned long integer (size is platform dependent).
 
-@item @qcode{"float"}
-Single precision floating point number (size is platform dependent).
 @end table
 
 @noindent
 The default precision is @qcode{"uchar"}.
 
 The @var{precision} argument may also specify an optional repeat
 count.  For example, @samp{32*single} causes @code{fread} to read
 a block of 32 single precision floating point numbers.  Reading in
